### ISSUE TITLE
Break timer like mobster (http://mobster.cc/) 

### ISSUE THREAD

**Issue by Austin117** [mrozbarry/mobtime#119]
Check out the break timer settings for the http://mobster.cc/ tool. Something like this would be very helpful. Thanks!

**Comment by dbartholomae** [comment]
Here's a proposal how to implement a first version:

- In the [timer settings](https://github.com/mrozbarry/mobtime/blob/dev/public/timer.js#L273) add a section called "breaks" which shows a [checkbox](https://github.com/mrozbarry/mobtime/blob/dev/public/components/checkbox.js)
which by default is not checked. If it is checked, another input line "time between breaks" appears which can be set to minutes similar to the "turn duration" at the beginning of the settings.
- If "time between breaks" is set, a new section appears in the [overview after the mob participants](https://github.com/mrozbarry/mobtime/blob/dev/public/timer.js#L208). This section is called "Break" and by default shows the text "Start turn to schedule next break". The section's heading has an "Edit break" link that leads to the timer settings.
- Given "time between breaks" is set, when I click "Start turn", the "Break" section shows a timer counting down until the next break
- When the break timer is up, the text in the "Break" section changes to "Break after next rotation".
- The next time the mob timer runs out or a rotation is manually triggered, [instead of the `timeRemaining` section](https://github.com/mrozbarry/mobtime/blob/dev/public/timer.js#L113) it says "Break" with a button on the right "Finish break" and the "Break" section in the overview shows "Time for a break!"
- When the "Finish break" button is clicked, the app gets set back to the original state: "Remaining time" with a "Start turn" button is shown, and the "Break" section on the overview shows "Start turn to schedule next break"

**Comment by dbartholomae** [comment]
I'll take a stab at it now

**Comment by alterisian** [comment]
Is the break timer enchancemet now in the live timer?

**Comment by dbartholomae** [comment]
No, I think @mrozbarry doesn't want to add it at the moment, see https://github.com/mobtimeapp/mobtime/pull/163#issuecomment-733096282

**Comment by dbartholomae** [comment]
I think this is a good explanation on the issue: https://github.com/mobtimeapp/mobtime/issues/120#issuecomment-846427923
